src/test/java/org/togetherjava/discordbot/util/DurationFormatterTest.java

```diff
@@ -0,0 +1,76 @@
+package org.togetherjava.discordbot.util;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.Duration;
+import static org.assertj.core.api.Assertions.*;
+class DurationFormatterTest {
+
+  private DurationFormatter durationFormatterUnderTest;
```

Comment:
This seems like a long variable name for no reason. A better name would be `formatter` or `durationFormatter`.